scripts/ntlm-methods.nse

```diff
@@ -0,0 +1,58 @@
+local http = require "http"
+local shortport = require "shortport"
+local stdnse = require "stdnse"
+local string = require "string"
+local table = require "table"
+
+description = [[
+Returns authentication methods a winrm server supports.
+]]
+
+---
+-- @usage
+-- nmap --script winrm-auth-methods <target>
+--
+-- @output
+-- 5985/tcp open  wsman
+-- | winrm-auth-methods: 
+-- |   Accepted Authentication Methods: 
+-- |     Negotiate
+-- |     Basic
+-- |     Kerberos
+-- |_    CredSSP
+
+author = "Evangelos Deirmentzoglou"
+license = "Same as Nmap--See https://nmap.org/book/man-legal.html"
+categories = {"default", "discovery", "safe"}
+
+portrule = shortport.port_or_service(5985, 5986)
```

Comment:
The port numbers need to be in a table, otherwise this means "the port 5985 or the service '5986'." Probably use shortport.portnumber instead.